api/api/models/media.py

```diff
@@ -65,19 +70,37 @@ class AbstractMedia(
         "Source and provider can be different. Eg: the Google Open "
         "Images dataset is source=openimages, but provider=flickr.",
     )
-    last_synced_with_source = models.DateTimeField(blank=True, null=True, db_index=True)
-    removed_from_source = models.BooleanField(default=False)
-
-    view_count = models.IntegerField(
+    last_synced_with_source = models.DateTimeField(
         blank=True,
         null=True,
-        default=0,
+        db_index=True,
+        help_text="The date the media was last updated from the upstream source.",
+    )
+    removed_from_source = models.BooleanField(
+        default=False,
+        help_text="Whether or not the media has been removed from the upstream source.",
+    )
+
+    view_count = models.IntegerField(
+        blank=True, null=True, default=0, help_text="Vestigial field, purpose unknown."
```

Comment:
Should we leave it out then? It's odd to include such a thing in public documentation, especially in a versioned API...

Reply:
When you say leave it out, do you mean not include any documentation for it? Since this documentation is also for us as maintainers, I think having this kind of information is useful for us too.

Reply:
Yes, sorry, I had the wrong context in mind for this. Resolving.